sys-apps/fwts/fwts-24.01.00.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="https://github.com/fwts/fwts/archive/refs/tags/V${PV}.tar.gz -> ${P}.ta
 LICENSE="GPL-2"
```

Comment:
probably should be `GPL-2+`, see: https://github.com/fwts/fwts/blob/master/src/main.c#L6-L7